src/main/java/Duke.java

```diff
@@ -26,28 +26,56 @@ static void endSession() {
 
     static void storeTasks(ArrayList<Task> Tasks, String text) {
         System.out.println(linebreak);
-        Task newTask = new Task(text);
-        Tasks.add(newTask);
-        System.out.println("added: " + text);
+        String[] result = text.split(" ");
+        String[] result2 = text.split("/by ");
+        String[] result3 = text.split("/at ");
+        Todo newTodo = null;
+        Deadline newDeadline = null;
+        Event newEvent = null;
+        if (result[0].equals("todo")) {
+            newTodo = new Todo(result[1]);
+            Tasks.add(newTodo);
+        }
+        else if (result[0].equals("deadline")) {
+            newDeadline = new Deadline(result2[0].substring(9), result2[1]);
+            Tasks.add(newDeadline);
+        }
+        else if (result[0].equals("event")) {
+            newEvent = new Event(result3[0].substring(6), result3[1]);
+            Tasks.add(newEvent);
+        }
+        System.out.println("I have added this task!");
+        if (result[0].equals("todo")) {
+            System.out.println(newTodo);
+        }
+        else if (result[0].equals("deadline")) {
+            System.out.println(newDeadline);
+        }
+        else if (result[0].equals("event")) {
+            System.out.println(newEvent);
+        }
```

Comment:
You should be able to just print out tasks using a to string method instead of rechecking.